crates/musli-zerocopy/src/pointer/ref.rs

```diff
@@ -51,10 +50,47 @@ where
 {
     offset: O,
     metadata: T::Stored<O>,
-    #[zero_copy(ignore)]
     _marker: PhantomData<(E, T)>,
 }
 
+unsafe impl<T, E, O> ZeroCopy for Ref<T, E, O>
+where
+    T: ?Sized + Pointee,
+    E: ByteOrder,
+    O: Size,
+{
+    const ANY_BITS: bool = O::ANY_BITS && T::Stored::<O>::ANY_BITS;
```

Comment:
`ANY_BITS` always has to be `false`, since Ref can only inhabit certain bit patterns. Specifically the ones which allows it to be constructed through `Ref::try_from_parts`.